mainhandler/handlerequests.go

```
@@ -115,8 +136,16 @@ func (mainHandler *MainHandler) SetupContinuousScanning(ctx context.Context) err
 	}
 
 	fetcher := cs.NewFileFetcher(rulesReader)
+	if fetcher == nil {
+		panic("fetcher is nil")
```

We should return the error and handle it in the calling function.

---

Now _that_ is a leftover from debugging. Will fix.

---

Fixed.